sklearn/compose/_column_transformer.py

```diff
@@ -372,6 +373,53 @@ def _fit_transform(self, X, y, func, fitted=False):
             else:
                 raise
 
+    def _calculate_inverse_indices(self, X):
+        """
+        Private function to calcuate indicies for inverse_transform
+        """
+        # checks for overlap
+        all_indexes = set()
+        input_indices = []
+        for name, trans, cols in self.transformers:
+            col_indices = _get_column_indices(X, cols)
+            if not all_indexes.isdisjoint(set(col_indices)):
```

Comment:
I'm probably missing something. I can't see where you update all_indexes to be non-empty. If I'm right, add tests

Reply:
Good catch! It turns out `pytest.raises(..., match=...)` was need to properly test the Exception.